R/utils_render_html.R

```diff
@@ -752,11 +753,12 @@ create_body_component_h <- function(data) {
   all_default_vals <- unname(as.matrix(body[, default_vars]))
 
   alignment_classes <- paste0("gt_", col_alignment)
+
   if (stub_available) {
     n_cols <- n_data_cols + 1
     alignment_classes <- c("gt_left", alignment_classes)
     stub_var <- dt_boxhead_get_var_stub(data = data)
-    all_stub_vals <- as.matrix(body[, stub_var])
+    all_stub_vals <- as.matrix(stub_df[, "rowname"])
```

Comment:
What's the reason for this change? Does it work when the data column for the stub isn't called "rowname"?